### PR TITLE
Version update on code- please review

### PR DESCRIPTION
**Pull request by imran21**
I have done some changes in the code and provided the version number in the print statement. 
System.out.println( "This is on branch dev - v1.0" );
System.out.println( "Hope this version finds well" );

### CODE REVIEW

File: CreateWebSource.java
Comment (DevOpsImran) [inline CreateWebSource.java:1]: Please remove the branch dev and just have "This is on  - v1.0"

File: CreateWebSource.java [hunk 1]
```diff
@@ -19,6 +19,7 @@ public static void main(String[] args) throws Exception
   {
     System.out.println( "" );
     System.out.println( "This is on branch dev - v1.0" );
+    System.out.println( "Hope this version finds well" );
```
Comment (DevOpsImran) [inline CreateWebSource.java]: This Message need not to be there - I think.

**Review by DevOpsImran — CHANGES_REQUESTED** [review]

File: CreateWebSource.java (hunk 1 above)
Comment (imran21) [reply CreateWebSource.java]: This has been resolved

**Review by DevOpsImran — APPROVED** [review]

**Review by DevOpsImran — APPROVED** [review]

**Review by DevOpsImran — APPROVED** [review]
Approved

**Review by DevOpsImran — APPROVED** [review]

**Review by DevOpsImran — APPROVED** [review]

File: CreateWebSource.java [hunk 2]
```diff
@@ -19,8 +19,7 @@ public static void main(String[] args) throws Exception
   {
     System.out.println( "" );
```
Comment (DevOpsImran) [inline CreateWebSource.java]: What this is for?

**Review by DevOpsImran — CHANGES_REQUESTED** [review]

File: CreateWebSource.java (hunk 2 above)
Comment (imran21) [reply CreateWebSource.java]: This has been came along with code from beginning. Let me remove it.

**Review by DevOpsImran — APPROVED** [review]

**Review by imran21 — COMMENTED** [review]
This has been removed

**Review by DevOpsImran — APPROVED** [review]

**Review by DevOpsImran — APPROVED** [review]

**Review by DevOpsImran — CHANGES_REQUESTED** [review]
Is there any updates on project

**Comment by imran21** [comment]
> Is there any updates on project

yes we have

**Review by DevOpsImran — APPROVED** [review]

**Pull request merged**